showcase/src/assets/examples/table/with-custom-actions/table.ts

```diff
@@ -39,25 +28,27 @@ export class TableWithCustomActionsComponent implements OnInit {
 
 		this.columns = [
 			{ name: "id", label: "Id" },
-			{ name: "title", label: "Title", cellFormatter: (value: { label: string }): string => "~" + value.label },
-			{ name: "description", label: "Description" }
+			{
+				name: "title",
+				label: "Title",
+				cellFormatter: (value: { label: string }): string => "~" + value.label
+			},
+			{ name: "description", label: "SHOWCASE.DEMO.TABLE.LABELS.DESCRIPTION" }
```

Comment:
Can you replace the translation key (`SHOWCASE.DEMO.TABLE.LABELS.DESCRIPTION`) by the actual text: `Description`?